components/page-actions.tsx

```diff
@@ -245,7 +246,14 @@ export default function PageActions({
         !isWorkingOnBounty &&
         isRepoForked &&
         isStateToWorking)
-      return (
+
+      if(state.currentBounty?.data?.isKyc 
+          && state?.currentUser?.kyc 
```

Comment:
you dont need to check for `state?.currentUser?.kyc && state?.currentUser?.kyc?.status` since `?.` (null coalescent) should take care of that for you